projects/igniteui-angular/src/lib/paginator/page-size-selector.component.html

```diff
@@ -2,7 +2,6 @@
 <div class="igx-page-size__select">
     <igx-select [overlaySettings]="paginator.overlaySettings" [(ngModel)]="paginator.perPage"
         [displayDensity]="paginator.paginatorSelectDisplayDensity" type="border">
-        <label igxLabel [hidden]="true">{{ paginator.resourceStrings.igx_paginator_label }}</label>
```

Comment:
Why is this line being deleted?

Reply:
I can't see any reason for the label to be duplicated. This also causes UI problems.